test/factories/call_list.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :call_list do
+    user
+    patient
+    line { 'DC' }
+    sequence :order_key do |n|
+      n
+    end
```

Comment:
You don't need the block in this case, [docs](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#without-a-block)
```suggestion
    sequence (:order_key)
```